syntax = "proto3";
package object_micro_app;




import "google/protobuf/empty.proto";

/*
BindObjectMicroApp请求
*/
message BindObjectMicroAppRequest {
    /*
    小产品id
    */
    string appId = 1 ;
}

/*
BindObjectMicroAppApi返回
*/
message BindObjectMicroAppResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    google.protobuf.Empty data = 4 ;
}

